scss/forms/_form-select.scss

```diff
@@ -42,7 +42,6 @@
 
   &[multiple],
   &[size]:not([size="1"]) {
-    height: auto;
```

Comment:
@m5o, we should also revert this line.

Reply:
ok, will do – technically it was not my change 😉